pset-iterators.js

```diff
@@ -137,6 +165,14 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 5:')
 
 // Add your code below this line
+const livingPresidents = presidents.filter(function (president) {
+    return president.death_year === null
+  })
+
+console.log("Presidents who are still alive: ")
+console.log(livingPresidents)
```

Comment:
@AtleLohrmann Good Job here with this solution using the filter method to return the current living president's

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L148)